main.c

```diff
@@ -69,6 +69,22 @@ int main(void)
     MSS_TIM64_enable_irq();
     NVIC_SetPriority(Timer1_IRQn, 1);
 
+    /* load citi conf on startup */
+    uint32_t cfg_id_32 = 0;
+    mem_read(NVM_CITIROC_CONF_NO, &cfg_id_32);
+    uint8_t cfg_id = (uint8_t)(cfg_id_32);
+    uint8_t *nvm_cfg_addr =
+    (uint8_t *)(NVM_ADDR+CITIROC_OFFSET+(cfg_id*CITIROC_LEN));
+
+    if (cfg_id == nvm_cfg_addr[CITIROC_LEN-1]) {
+        mem_ram_write(RAM_CITI_CONF, nvm_cfg_addr);
+        citiroc_send_slow_control();
+    }
+
```

Comment:
What happens in case the config is not set here? Please add a `TODO` -- and it would be greatly appreciated if you could offer some thoughts in your report. (@anwestm I guess this is also to you.)

Reply:
It checks the id with the id in the configuration. if they don't match, the config will be applied. This should probably be handled anyways.

Reply:
Thanks for adding the TODO. It probably works as it is now in that no config will be applied to the ASIC, we should figure out in the future a means of informing "the ground" that no ASIC was applied.

As stated above, some thoughts (if you have any) on how this could be done would be greatly appreciated in your report!